frontend/.storybook/preview-head.html

```diff
@@ -0,0 +1,11 @@
+<style>
+	#root {
+		padding: 10px;
+	}
+	.storybook--lg {
+		min-width: 900px;
+	}
+	.storybook--sm {
+		max-width: 400px;
+	}
+</style>
```

Comment:
Just wondering if it would be better to put these in a `storybook.css` file and important them in the config.js.

Reply:
I noodled on that idea and honestly it was a coin-toss. I'm completely open to that approach. I'm gonna keep this as is for now to get master up and running. But I can make that change in a separate branch.

Reply:
cool. If you thought about both and landed on this, alls good